plugins/org.python.pydev.parser/tests/org/python/pydev/parser/PyParser38Test.java

```diff
@@ -141,4 +146,27 @@ public void testPositionalArgs4() {
         parseLegalDocStr("a = lambda a, b=1, /, c=2, d=3, *, e=4, f=5: None");
     }
 
+    private void checkStr(String s, String expected, boolean binary, boolean fstring, boolean unicode, boolean raw) {
+        SequencialASTIteratorVisitor visitor = SequencialASTIteratorVisitor.create(parseLegalDocStr(s));
+        List<ASTEntry> asList = visitor.getAsList(Str.class);
+        assertEquals(asList.size(), 1);
+        ASTEntry entry = asList.iterator().next();
+        Str str = (Str) entry.node;
+        assertEquals(binary, str.binary);
+        assertEquals(fstring, str.fstring);
+        assertEquals(unicode, str.unicode);
+        assertEquals(raw, str.raw);
+        assertEquals(expected, str.s);
+    }
+
+    public void testStr() throws Exception {
+        checkStr("''", "", false, false, false, false);
+        checkStr("'s'", "s", false, false, false, false);
+
+        checkStr("b''", "", true, false, false, false);
+        checkStr("b's'", "s", true, false, false, false);
+
+        checkStr("f''", "", false, true, false, false);
+        checkStr("f's'", "s", false, true, false, false);
```

Comment:
It's missing the tests for what was just added. i.e.: `rf'xxx'` and `rb'xxx'`.

Reply:
(i.e.: my previous comment was just about how to structure the tests, not all the tests that should be done -- it doesn't need to be exhaustive but it should at least check for the new features).

Reply:
Thats right.. I completely passed by it. My bad

Reply:
One thing that I've just checked is that you can also use `br` and not just `rb`. Can you check that too?

Reply:
Oops, it's actually there.